### ISSUE TITLE
About the training of contextual embedding

### ISSUE THREAD

**Issue by liangpingding** [allanj/ner_incomplete_annotation#10]
Hi, thanks for sharing the code! I saw that it's possible to use contextual embedding such as elmo. And in your code of main.py, do you train elmo by yourself?  I aslo saw that there are three elmo vector files, do you train elmo for each train/dev/test set? If you can provide some reference code for training elmo models, that would give us more details!

**Comment by allanj** [comment]
I did not train ELMo but using existing pretrained ELMo model from different languages. Usually people pretrain them following the code add provided by AllenNLP, let me know if you need more information
